CHANGELOG.md

```diff
@@ -4,6 +4,33 @@ All notable changes to this project will be documented in this file.
 Each new release typically also includes the latest modulesync defaults.
 These should not affect the functionality of the module.
 
+## [v9.0.0](https://github.com/voxpupuli/puppet-grafana/tree/v9.0.0) (2021-08-16)
+
+[Full Changelog](https://github.com/voxpupuli/puppet-grafana/compare/v8.0.0...v9.0.0)
+
+**Breaking changes:**
+
+- Support only 6.x/7.x/8.x versions [\#246](https://github.com/voxpupuli/puppet-grafana/pull/246) ([root-expert](https://github.com/root-expert))
```

Comment:
I think https://github.com/voxpupuli/puppet-grafana#module-description should be updated to reflect this before we release.

Reply:
Probably `Limitations` section as well

Reply:
updated it. please have a look.